app/scripts/modules/google/loadBalancer/configure/internal/gceCreateInternalLoadBalancer.controller.ts

```diff
@@ -34,7 +34,6 @@ class InternalLoadBalancer implements IGceLoadBalancer {
   account: string;
   network: string = 'default';
   subnet: string;
-  region: string;
```

Comment:
Yes.  It was also being included in the ctor which creates a local variable in TS so the compiler was complaining.